vignettes/Vaccines.Rmd

```diff
@@ -40,153 +123,262 @@ simparams <- get_parameters(list(
+First, we will model a single round of RTS,S vaccine given in a mass vaccination strategy where those within a given age range will be vaccinated to a specified coverage level and during a short window of time. In this example, individuals aged between 5 months and 10 years are vaccinated with a primary series (3 doses) followed by a single booster 12 months after the initial vaccination. The age groups to be vaccinated as well as timing of all vaccinations can be modified within the `set_mass_rtss()` function.
 
-```{r}
-rtssparams <- simparams
+We specify that the intervention should use the RTS,S vaccine through the `profile` and `booster_profile` inputs.
 
+```{r}
 rtssparams <- set_mass_rtss(
-  rtssparams,
-  timesteps = 1 * year,
-  coverages = 1,
-  min_wait = 0,
-  min_ages = 5 * month,
-  max_ages = 10 * year,
-  boosters = 18 * month,
-  booster_coverage = 0.95
+  simparams,
+  timesteps = 1 * year, # The single round of vaccination is at 1 year into the simulation.
+  coverages = 1, # The vaccine is given to 100% of the population between the specified ages.
+  min_wait = 0, # The minimum acceptable time since the last vaccination is 0.
+  min_ages = 5 * month, # The minimum age for the target population to be vaccinated. 
+  max_ages = 10 * year, # The maximum age for the target population to be vaccinated.
+  boosters = 12 * month, # The booster is given at 12 months after the initial vaccination. 
+  booster_coverage = 0.95 # Coverage of the booster dose is 95%.
 )
 
 output <- run_simulation(sim_length, rtssparams)
-output$clinical_incidence <- 1000 * output$n_inc_0_1825 / output$n_0_1825
+```
 
-ggplot(data = output, aes(x = timestep / 365, y = clinical_incidence)) + 
-  geom_line(col = "grey80") +
-  stat_smooth(col = "darkblue", se = FALSE) +
-  geom_vline(xintercept = 1, col = "red") +
-  xlab("year") +
-  ylab("Clinical incidence \n (per 1000 children aged 0-5)") +
-  theme_bw()
+#### Plot clinical incidence and prevalence
 
+```{r, fig.width = 8, fig.height = 4, fig.align = 'center'}
+par(mfrow = c(1,2))
+plot_inci()
+plot_prev()
 ```
+We see a much more gradual decrease in clinical incidence following EPI implementation compared to the mass vaccination strategy.  
+  
+#### Plot doses
+
+You can look at the distribution of doses using the `n_rtss_mass_dose_*` or `n_rtss_epi_dose_*` outputs. It is always a good idea to check the timing of vaccine doses to ensure that it is specified as intended.
+
+```{r, fig.width = 6, fig.height = 4, fig.align = 'center', message=FALSE}
+plot_doses()
+```
+
+### Seasonal mass vaccination
+
+Mass vaccination can also be targeted seasonally. For example, we may want to have a mass vaccination campaign a few months prior to the peak transmission season. In the example below, we will create a parameter set with seasonality and first run a simulation with no vaccine, then run a simulation with a mass vaccination campaign targeting everyone between the ages of 5 months and 15 years.
```

Comment:
... we may want to have a mass vaccination campaign take place a few months prior ...